go-apps/meep-metrics-engine/server/api_subscriptions.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
move generated code to version-specific folders

Reply:
fixed